hieradata_aws/class/integration/account_api_db_admin.yaml

```diff
@@ -1,16 +1,4 @@
 govuk_env_sync::tasks:
```

Comment:
I think this line will need commenting out too (or the commented out task being set to `ensure: "disabled"`) otherwise this YAML value will be set to null rather than an empty object - not 100% sure if that actually causes problems though.